src/vm/eepolicy.cpp

```diff
@@ -1200,6 +1200,14 @@ inline void DoLogForFailFastException(LPCWSTR pszMessage, PEXCEPTION_POINTERS pE
         {
             PrintToStdErrA("\n");
             LogCallstackForLogWorker();
+
+            if (innerExceptionString != NULL) {
+                PrintToStdErrA("\n");
+                PrintToStdErrA("Inner exception details:");
```

Comment:
I don't think this exception should be described as the 'Inner Exception'. Its just the 'Exception'. Inner Exception implies that there is another exception X, and X.InnerException = this_exception, but that isn't the case here.